### ISSUE TITLE
Message context menu is not accessible

### ISSUE THREAD

**Issue by turt2live** [vector-im/riot-web#10899]
Reportedly it appears at the bottom of the page, which means that non-advanced screen reader users may miss it.

Might be as simple as moving the container higher? Or just stealing focus?

**Comment by ara4n** [comment]
i think the point is that it is appended to the DOM

**Comment by turt2live** [comment]
The lack of focus means they don't get read out at all - manipulating the DOM doesn't really affect the screen reader much, as long as all the other ducks are in a row.

**Comment by turt2live** [comment]
Leaving this open until confirmation of fix received: see https://github.com/matrix-org/matrix-react-sdk/pull/3454#issuecomment-532418564

**Comment by turt2live** [comment]
The fix applied is okay for now (see https://github.com/matrix-org/matrix-react-sdk/pull/3454#issuecomment-532938577 ) - closing in favour of https://github.com/vector-im/riot-web/issues/10917
